distributed/utils_test.py

```diff
@@ -1595,24 +1596,67 @@ def check_thread_leak():
             assert False, (bad_thread, call_stacks)
 
 
-@contextmanager
-def check_process_leak(check=True):
-    for proc in mp_context.active_children():
+def wait_active_children(timeout: float) -> list[multiprocessing.Process]:
+    """Wait until timeout for mp_context.active_children() to terminate.
+    Return list of active subprocesses after the timeout expired.
+    """
+    t0 = time()
```

Comment:
Tiny super nit: `monotonic` (or perhaps `perf_counter`) would technically be more appropriate here I think. I'm sure it will never ever make a difference in this case, and there are plenty of other more important places where we already use non-monotonic `time` and shouldn't. I'd just like to start being more careful about it.

Reply:
Something for a broader discussion maybe? If we want to switch to monotonic, it should be done consistently everywhere, e.g. replace metrics.time. As usual, Windows is a source of pain so need to thread carefully there.